tests/Functional/src/com/microsoft/aad/adal/test/DefaultTokenCacheStoreTests.java

```diff
@@ -250,4 +264,116 @@ public void testExpireBuffer() {
+        TokenCacheItem frtTokenCacheItem =(TokenCacheItem)m.invoke(store, userId);
+        assertTrue(frtTokenCacheItem == null);
+    }    
+    
+    public void testSerialize_invalidUserId() throws ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException{
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        Method m = ReflectionUtils.getTestMethod(store, "serialize", String.class);
+        
+        try {
+            String jsonString = (String)m.invoke(store, "");
+            Assert.fail("not expected");
+        } catch (InvocationTargetException exp){
+             assertTrue(exp.getCause() instanceof IllegalArgumentException);
+        }
+        
+        String testStr = null;
+        try {
+            String jsonString = (String)m.invoke(store, testStr);
+            Assert.fail("not expected");
+        } catch (InvocationTargetException exp){
+            assertTrue(exp.getCause() instanceof IllegalArgumentException);
+        }        
+    }
+    
+    public void testSerialize_nullCacheItem() throws AuthenticationException, IllegalAccessException, IllegalArgumentException, InvocationTargetException, ClassNotFoundException, NoSuchMethodException, InstantiationException{
+        String userId = "userid2";
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        Method m = ReflectionUtils.getTestMethod(store, "serialize", String.class);        
+        assertTrue((String)m.invoke(store, userId) == null);
+    }
+     
+    public void testSerialize_valid() throws AuthenticationException, IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        String userId = "userid2";
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        addFRTCacheItem(store);
+        Method m = ReflectionUtils.getTestMethod(store, "serialize", String.class);
+        String jsonStr = (String)m.invoke(store, userId);
+        assertTrue( jsonStr != null);
+    }
+
+    public void testDeserialize_valid() throws AuthenticationException, IllegalArgumentException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        String userId = "userid2";
+        DefaultTokenCacheStore store = (DefaultTokenCacheStore)setupItems();
+        addFRTCacheItem(store);
+        Method m = ReflectionUtils.getTestMethod(store, "serialize", String.class);        
+        String serializedBlob = (String)m.invoke(store, userId); 
+        
+        Method deserializeReflect = ReflectionUtils.getTestMethod(store, "deserialize", String.class);
+        deserializeReflect.invoke(store, serializedBlob);
+        
+        Method getfrtReflect = ReflectionUtils.getTestMethod(store, "getFamilyRefreshTokenItemForUser", String.class);
```

Comment:
IMO, should do a simple cache look up here and verify the item is in the cache and it is as expected.